### DISCUSSION TITLE
Error con JAVA_PATH al instalar Maven

### DISCUSSION THREAD

**Discussion by gastonbaeza** [dds-jv/foro#99] [Consultas de herramientas] [answered]
Buenas!
Tengo Windows 10 y la version de java 1.8.0_241. Instalé Maven según el tutorial (_https://maven.apache.org/install.html_) pero al intentar ejecutarlo me tira un error en el JAVA_HOME:

![image](https://user-images.githubusercontent.com/26452827/78461587-d47c9200-76a0-11ea-8d04-0c0fc1fbb475.png)

Esa variable está configurada de la siguiente manera:

![image](https://user-images.githubusercontent.com/26452827/78461631-3210de80-76a1-11ea-8cdc-084719144159.png)

Y tal como lo pide el tutorial de instalación, agregé al path la carpeta bin de maven (en **user variables** y **system variables**):

![image](https://user-images.githubusercontent.com/26452827/78461702-afd4ea00-76a1-11ea-8e53-e671429b187f.png)

No sé qué me puede estar faltando, o no me doy cuenta qué estoy haciendo mal. 
Saludos!

**Comment by BrianLuna** [comment]
Hola Gaston, en la variable path, luego de C:\Users\gabaeza\Documents\maven\apache-maven-3.6.3\bin, agregá ";%JAVA_HOME%\bin" (sin las comillas).

Lo que entiendo que te está pasando es que en la variable path tenés que tener referenciada la carpeta "bin" del directorio de instalación del JDK de Java, y no la estás referenciando.

Avisame si eso sirve.

**Comment by gastonprieto** [comment]
puede ser que sea el ; final de la ruta del JAVA_HOME también

creo que lo que vos decís @BrianLuna el lo tiene pero con un path absoluto en vez de usando variables de entorno.

**Comment by BrianLuna** [comment]
Lo que veo es que en la variable path solo tiene la dirección absoluta de maven, no de java...

**Comment by BrianLuna** [comment]
No dije nada!! No vi lo de arriba

**Comment by gastonbaeza** [comment]
@BrianLuna en vez de referenciarla con la variable %JAVA_HOME%, lo hago directamente con **C:\Program Files\Java\jdk1.8.0_241\bin**, se ve en la captura.

y @gastonprieto eso siempre me hizo ruido, pero yo no tengo ese ";" al final en la variable:
![image](https://user-images.githubusercontent.com/26452827/78462515-45c04300-76a9-11ea-8b07-9005b783061c.png)

**Comment by gastonprieto** [comment]
Mmm, probaría con el paso 4 de esta guía: https://javatutorial.net/set-java-home-windows-10

por otro lado tal vez te esta afectando que en `PATH` tenes tanto la jre como la jdk

**Comment by gastonbaeza** [comment, marked as answer]
Era el punto y coma!
Por más que editaba o borraba y volvía a agregar la ruta al JAVA_HOME, al final me ponía ese **;**.
Lo resolví haciendo doble click en JAVA_HOME pero en vez de seleccionar **New** o **Edit**, fui a **Edit Text...** y ahí vi el punto y coma que sobraba, lo borré y listo. 
Gracias a ambos!
